web/styles/ant-overrides.scss

```diff
@@ -160,6 +160,7 @@ DROPDOWN
   height: 100%;
   .ant-tabs-content-holder {
```

Comment:
This looks like it would add 8px of margin under every use of tabs, anywhere. Instead, it would be better to only add spacing to just this one layout.